static/homepage.js

```diff
@@ -0,0 +1,50 @@
+(function () {
+  var rows = [["Days Before", "General", "Runoff"]];
```

Comment:
```suggestion
  var rows = [["Days Before", "General 2020", "Runoff 2021"]];
```

Nice use of C3! I wasn't sure if General and Runoff was that clear, just a suggestion though, your call